docs/workflows/lambda.md

```diff
@@ -12,7 +12,7 @@ AWS provides great getting started guide for building Lambdas in the [developer
 
 Cumulus currently supports the following environments for Cumulus Message Adapter enabled functions:
 
-* [Node.js 10.16.3](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
+* [Node.js 10.x, 12.x](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
 * [Java 8](https://docs.aws.amazon.com/lambda/latest/dg/java-programming-model.html)
 * [Python 2.7, 3.6](https://docs.aws.amazon.com/lambda/latest/dg/python-programming-model.html)
```

Comment:
This isn't your PR, but we expressly support only python 3.6+ (3.6-3.8 aws lambda runtimes) for python lambdas.